integration_tests/e2e/refer.cy.ts

```diff
@@ -42,14 +42,20 @@ context('Refer', () => {
     startReferralPage.shouldContainStartButtonLink()
   })
 
-  it("Shows the 'find person' page for a referral", () => {
+  it("Allows users to search for a person and confirm that person's details", () => {
```

Comment:
So are we favouring testing two pages together where they combine to form a discreet part of the overall journey? I guess this makes it easier to test the search action without duplicating most of the find person page logic in a discreet test

I suppose though I'll end up needing to duplicate some of the confirm person page logic from this test when testing the create and referral show actions in my PR?

Reply:
(Or, more concisely: we will have a lot of on-click/between page functionality - so I wonder how we cut up our integration tests)

Reply:
Yeah, integration test-wise, I think it's useful to chop things up into smaller discreet parts as you've mentioned, rather than testing a whole user journey. In our E2E tests we can test the joined up happy path for the whole journey.

For the create/show part of your PR, I think we could visit the confirm person page for a person, not bother about asserting anything, then click "Continue" and assert that see the created Referral.

Reply:
Yeah, I think that's where I was heading. Thanks!